sympy/utilities/tests/test_lambdify.py

```diff
@@ -460,6 +469,24 @@ def test_numexpr_userfunctions():
     func = lambdify((x, y), uf(x, y), modules='numexpr')
     assert numpy.allclose(func(a, b), 2*a*b+1)
 
+def test_tensorflow_basic_math():
+    if not tensorflow:
+        skip("tensorflow not installed.")
+    expr = Max(sin(x), Abs(1/(x+2)))
+    func = lambdify(x, expr, modules="tensorflow")
+    a = tensorflow.constant(0, dtype=tensorflow.float32)
+    s =  tensorflow.Session()
+    assert func(a).eval(session=s) == 0.5
+
+def test_tensorflow_placeholders():
+    if not tensorflow:
+        skip("tensorflow not installed.")
```

Comment:
Gotcha. In that case, is the way it's setup now okay? We test in `.travis.yml` to use whichever Python version is already assigned to `$TRAVIS_PYTHON_VERSION`, which I assume is the Python version already on the box.